src/Tizen.Multimedia.Vision/MediaVision/FaceDetector.cs

```diff
@@ -103,5 +109,89 @@ private static InteropFace.DetectedCallback GetCallback(TaskCompletionSource<Rec
                 }
             };
         }
+
+        /// <summary>
+        /// Detects faces on the source image using inference engine set in <paramref name="config"/>.<br/>
+        /// Each time when DetectAsync is called, a set of the detected faces at the media source are received asynchronously.
+        /// </summary>
+        /// <remarks>
+        /// If there's no detected face, empty collection will be returned.
+        /// </remarks>
+        /// <feature>http://tizen.org/feature/vision.inference</feature>
+        /// <feature>http://tizen.org/feature/vision.inference.face</feature>
+        /// <param name="source">The source of the media where faces will be detected.</param>
```

Comment:
Suggestion:

        /// <param name="source">The media source where faces are detected.</param>